### PR TITLE
feature #3845 - added edcon to public chats and using keyboard avoidi…

### LINKED ISSUE #3845: Add edcon to the public list of channels in status
For the upcoming attendance of edcon would like to include the public channel in status.

### PR DESCRIPTION
**Pull request by goranjovic**
fixes #3845

### Summary:

The same thing as #3844 only signed 

status: ready

### CODE REVIEW

**Review by pablanopete — APPROVED** [review]
Lgtm

File: src/status_im/ui/screens/add_new/new_public_chat/view.cljs [hunk 1]
```diff
@@ -51,12 +51,12 @@
...
                       :icon-opts {:color :gray}}]]]])
 
-(def default-public-chats ["status" "openbounty" "ethereum"])
+(def default-public-chats ["status" "openbounty" "edcon" "ethereum"])
 
 (views/defview new-public-chat []
   (views/letsubs [topic [:get :public-group-topic]
                   error [:get :public-group-topic-error]]
-    [react/view styles/group-container
+    [react/keyboard-avoiding-view styles/group-container
```
Comment (jeluard) [inline src/status_im/ui/screens/add_new/new_public_chat/view.cljs:59]: Does this fix some other issue?

**Review by jeluard — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/add_new/new_public_chat/view.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/add_new/new_public_chat/view.cljs:59]: yes, annoying one

Comment (flexsurfer) [reply src/status_im/ui/screens/add_new/new_public_chat/view.cljs:59]: but we need this new topic for conference , let's say we need this fix for conference too :)

**Comment by yevh-berdnyk** [comment]
Android: https://i.diawi.com/9q1TMG
iOS: https://i.diawi.com/hPo7x2

LGTM

<blockquote><div><strong><a href="https://i.diawi.com/9q1TMG">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>
<blockquote><div><strong><a href="https://i.diawi.com/hPo7x2">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Pull request merged**